app/src/panels/metric/metric.vue

```diff
@@ -103,10 +103,13 @@ export default defineComponent({
 						fields: [props.field],
 					},
 				});
-
 				if (props.field) {
 					if (props.function === 'first' || props.function === 'last') {
-						metric.value = Number(res.data.data[0][props.field]);
+						if (typeof res.data.data[0][props.field] === 'string') {
+							metric.value = res.data.data[0][props.field];
+						} else {
+							metric.value = Number(res.data.data[0][props.field]);
+						}
```

Comment:
I think we can rework this a hair to make it a little more obvious what's going on

Reply:
Nevermind, there's just a lot of branching paths here. lgtm for now